avocado/core/test.py

```diff
@@ -383,6 +384,11 @@ def _start_logging(self):
                                                        formatter,
                                                        self._ssh_logfile)
 
+        if isinstance(sys.stdout, output.LoggingFile):
+            sys.stdout.add_logger(logging.getLogger("avocado.test.stdout"))
+        if isinstance(sys.stderr, output.LoggingFile):
+            sys.stderr.add_logger(logging.getLogger("avocado.test.stderr"))
+
```

Comment:
Actually it looks like it's just timing issue, because if I remove the fix and only log that it would have been used (if present), it logs the presence of the condition, but the test proceeds without any issues...

https://travis-ci.org/avocado-framework/avocado/jobs/178775846